.github/workflows/static.yml

```diff
@@ -4,7 +4,7 @@ name: Static analysis
 jobs:
   phpstan:
     name: PHPStan
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
The issue with `ubuntu-latest` is that we get warnings when github changes the distribution for this value. 

Ie, 6 months prior Github switched from `ubuntu-latest` meaning `ubuntu-18.04` to `ubuntu-20.04` the CI was all filled with warnings.

Reply:
Good point, I changed to ubuntu-20.04